pkg/vm/engine/disttae/txn_table.go

```diff
@@ -1533,12 +1533,17 @@ func (tbl *txnTable) newReader(
 	}
 	for i := 1; i < readerNumber; i++ {
 		if i == readerNumber-1 {
+			//var dst []ModifyBlockMeta
+			dst := make([]ModifyBlockMeta, len(blks[(i-1)*step:]))
```

Comment:
This loop is n*n?

Reply:
never mind, it is a lop with steps.

Still, what is the cost of ModifyBlockMeta.copy()?   Not sure if it is a problem...

Reply:
there' re only a  limited number of blks  whose deletes is empty.

Reply:
and next pr will remove these codes.